.travis.yml

```diff
@@ -44,4 +47,4 @@ notifications:
     on_failure: always
     on_start: false
 after_success:
-  coveralls --rcfile=.coveragerc
+  coveralls --rcfile=.coveragerc
```

Comment:
you removed the newline

Reply:
Does it metter here, as it is not .py file!!!